SCClassLibrary/Common/GUI/Base/QView.sc

```diff
@@ -89,7 +89,7 @@ View : QObject {
 		this.background = color;
 	}
 
-	backgroundImage_ { arg image; this.setBackgroundImage(image) }
+	backgroundImage_ { arg image, tileMode=1, alpha=1.0, fromRect; this.setBackgroundImage(image, tileMode, alpha, fromRect) }
```

Comment:
`setter_` methods should only take a single argument.

Reply:
fixed